lte/gateway/python/magma/pipelined/app/inout.py

```diff
@@ -190,6 +199,9 @@ def _install_default_egress_flows(self, dp, mac_addr: str = "", vlan: str = ""):
         Raises:
             MagmaOFError if any of the default flows fail to install.
         """
+
+        self._install_proxy_flows(dp)
```

Comment:
I don't think this will work for CWF(you're using tunnel ids from activate flows req and maybe other things not in CWF) so lets just put it under ` if self.config.setup_type == 'LTE':`